Wyam.Core/Modules/CopyFiles.cs

```diff
@@ -106,7 +106,7 @@ public IEnumerable<IDocument> Execute(IReadOnlyList<IDocument> inputs, IExecutio
                             .Select(file =>
                             {
                                 string destination = _destinationPath == null
-                                    ? Path.Combine(context.OutputFolder, PathHelper.GetRelativePath(Path.GetDirectoryName(path), Path.GetDirectoryName(file)), Path.GetFileName(file)) 
+                                    ? Path.Combine(context.OutputFolder, PathHelper.GetRelativePath(context.InputFolder, Path.GetDirectoryName(file)), Path.GetFileName(file)) 
```

Comment:
That makes sense, especially since we'll have to convert it eventually any way.

In fact, one thing I want to look at soon is a better abstraction for the file system to take the place of System.IO usages. I especially like the one used in Cake, so may take some inspiration there.